shlr/meson.build

```diff
@@ -1,12 +1,99 @@
-res = run_command(py3_exe, '-c', '__import__("sys").exit(__import__("os").path.exists("capstone"))')
-if res.returncode() == 0
-  if not git_exe.found()
-    error('Cannot load capstone library. Either provide capstone in ./shlr/capstone or install git, so it can be downloaded')
+capstone_dep = dependency('capstone', version: '>=3.0.4', required: false)
+if not capstone_dep.found() or not get_option('use_sys_capstone')
+  message('Use bundled capstone')
+
+  res = run_command(py3_exe, '-c', '__import__("sys").exit(__import__("os").path.exists("capstone"))')
+  if res.returncode() == 0
+    if not git_exe.found()
+      error('Cannot load capstone library. Either provide capstone in ./shlr/capstone or install git, so it can be downloaded')
+    endif
+
+    message('Cloning capstone')
+    git_cmd = 'clone -b next --depth 10 https://github.com/aquynh/capstone.git capstone'
```

Comment:
But why do you need `--depth 10`? Why not `1` instead?

Reply:
no idea, i didn't even look at the commands because they were already there